package.json

```diff
@@ -126,6 +129,8 @@
 		"@wordpress/prettier-config": "file:packages/prettier-config",
 		"@wordpress/project-management-automation": "file:packages/project-management-automation",
 		"@wordpress/scripts": "file:packages/scripts",
+		"appium": "1.17.1",
+		"babel-jest": "24.9.0",
```

Comment:
Shouldn't the version of `babel-jest` align with the version of `jest`?

Reply:
As far as I can tell, it should be the same version that `@wordpress/scripts` defines.

Reply:
I've it updated to 25.3.0 which is the one `jest` currently has 👍

Reply:
Yes, makes sense to align.